### PR TITLE
Add accessibility resources page

### LINKED ISSUE #54: Broken link
https://dfe-analytical-services.github.io/analysts-guide/RAP/rap.html#Clean_final_code from https://github.com/dfe-analytical-services/analysts-guide/blob/c36895b08774e45125f60fd2cf0739d17d6c5da8/learning-development/sql.qmd#L45 is broken.

### PR DESCRIPTION
**Pull request by cjrace**
## Overview of changes

Add a new accessibility tools and guidance page in the learning resources section

## Why are these changes being made?

Request by an analyst for a list of free tools to get started with, expanded into making this page so it's available for everyone and we can build it up over time too.

## Detailed description of changes

1. New accessibility resources page
2. Updated renv snapshot to latest R version
3. Fix broken link to RAP pages from SQL page

## Issue ticket number/s and link

Fixes #54 

## Checklist before requesting a review
- [x] I have checked the contributing guidelines
- [x] I have checked for and linked any relevant issues that this may resolve
- [x] I have checked that these changes build locally
- [x] I understand that if merged into main, these changes will be publicly available

### CODE REVIEW

**Comment by cjrace** [comment]
[Add quick notes for free accessibility tools and checking to analysts guide](https://trello.com/c/ZpXXypEZ/1473-add-quick-notes-for-free-accessibility-tools-and-checking-to-analysts-guide)

File: learning-development/accessibility.qmd [hunk 1]
```diff
@@ -4,33 +4,67 @@ title: "Accessibility"
 
 <p class="text-muted">Some quick links and tips for increasing awareness of accessibility in digital content</p>
 
+
+---
+
+## Accessibility in DfE
+
+DfE have our own [Digital Accessibility Hub](https://educationgovuk.sharepoint.com/sites/lvewp00043) filled with helpful information on making the digital world more accessible.
```
Comment (jen-machin) [inline learning-development/accessibility.qmd:12]: Potentially add in some context here around why it's important e.g. WCAG (and a link to latest WCAG)

File: learning-development/accessibility.qmd [hunk 2]
```diff
@@ -4,33 +4,67 @@ title: "Accessibility"
...
+
+There are also the [DfE design manual](https://design.education.gov.uk), which has been created by the digital teams at DfE and includes [DfE's digital accessibility guidance](https://design.education.gov.uk/accessibility).
```
Comment (jen-machin) [inline learning-development/accessibility.qmd:14]: There is also that cool thing that gives you numbers of users that are likely to have specific needs which was built by someone in DfE, it is in the design manual but might be nice to mention it separately https://design.education.gov.uk/tools/how-many-users

File: learning-development/accessibility.qmd [hunk 3]
```diff
@@ -4,33 +4,67 @@ title: "Accessibility"
...
+## Colour accessibility tools
+
+Contrast:
+
+- [WebAim (comparing two colours)](https://webaim.org/resources/contrastchecker/)
+- [Colour contrast matrix (comparing multiple colours](https://jansensan.github.io/color-contrast-matrix/)
+
+Colour blindness:
+
+- [Adobe colour accessibility (gives a pass / fail)](https://color.adobe.com/create/color-accessibility)
```
Comment (jen-machin) [inline learning-development/accessibility.qmd:27]: This doesn't work on desktop for me

File: learning-development/accessibility.qmd [hunk 4]
```diff
@@ -4,33 +4,67 @@ title: "Accessibility"
...
+- You can often check colours in grey scale by using a print preview and switching the colour mode
+- [ColToGrey function within the DescTools R package](https://andrisignorell.github.io/DescTools/reference/ColToGrey.html)
+
+For more advice about colour in charts and visualisations specifically, including more tools and resources for checking colours yourself, see the [Analysis function colours in charts guidance](https://analysisfunction.civilservice.gov.uk/policy-store/data-visualisation-colours-in-charts/#section-9).
+
+---
+
+## Free tools for reviewing web pages
+
+- [axe DevTools](https://chromewebstore.google.com/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?pli=1), free chrome extension and paid versions, we recommend this (and GDS tend to recommend too)
```
Comment (jen-machin) [inline learning-development/accessibility.qmd:42]: Chrome has a little c here and big C below

File: learning-development/accessibility.qmd [hunk 5]
```diff
@@ -4,33 +4,67 @@ title: "Accessibility"
...
-	- Colour blindness -
-	- Colour blindness simulation - 
-	- Greyscale - 
+No automated tools fully cover accessibility, manual testing is always required, so if you want to do manual testing have a look at testing out the assistive software commonly used by users yourself.
 
-Helpful browser addins
+Magnifier:
 
-These 'bookmarklets' are bookmarks that instead of saving a URL to a page, save a piece of Javascript code that executes on the page you are looking at, usually to highlight specific mark up. Very nifty!
+- Built into Windows ([guidance for how to turn on Magnifier](https://support.microsoft.com/en-gb/windows/use-magnifier-to-make-things-on-the-screen-easier-to-see-414948ba-8b1c-d3bd-8615-0e5e32204198#:~:text=To%20quickly%20turn%20on%20Magnifier,turn%20on%20the%20Magnifier%20switch.))
```
Comment (jen-machin) [inline learning-development/accessibility.qmd:56]: the way this is phrased makes it a bit confusing as there are two separate things under the Magnifier heading and one of them is actually called Magnifier and one isn't

File: learning-development/accessibility.qmd [hunk 6]
```diff
@@ -4,33 +4,67 @@ title: "Accessibility"
...
+- [Dragon](https://www.nuance.com/en-gb/dragon.html), available through the [experience lab in DfE](https://educationgovuk.sharepoint.com/sites/lvewp00043/SitePages/Experience-Lab.aspx)
 
-Voice activation
-	- Dragon is a paid for licence, though available through the empathy lab
+Along with devices and the software mentioned above, the [experience lab in the Sheffield DfE office](https://educationgovuk.sharepoint.com/sites/lvewp00043/SitePages/Experience-Lab.aspx) also has a range of other equipment available, including access to a set of vision emulating glasses that you can wear to emulate different visual impairments.
 
-Magnifier
-Built into windows? Check
\ No newline at end of file
+---
```
Comment (jen-machin) [inline learning-development/accessibility.qmd:70]: I'd like to see something about cognitive load in here somewhere but doesn't need to be you that adds it. This is a nice and very detailed resource about cognitive load and data visualisations though: https://nightingaledvs.com/cognitive-load-as-a-guide-12-spectrums-to-improve-your-data-visualizations/

**Review by jen-machin — COMMENTED** [review]
Some comments to look over!

File: learning-development/accessibility.qmd (hunk 2 above)
Comment (cjrace) [reply learning-development/accessibility.qmd:14]: Ah of course, I wish I'd dug this out for the DISD weekly actually, it's a helpful one for context and helping to share the scale of users impacted! Will add in

File: learning-development/accessibility.qmd (hunk 3 above)
Comment (cjrace) [reply learning-development/accessibility.qmd:27]: Weird, looks like just adding a '/' at the end fixes it, not sure how I got the wrong link though will push the fix shortly

File: learning-development/accessibility.qmd (hunk 1 above)
Comment (cjrace) [reply learning-development/accessibility.qmd:14]: Made this change in my PR comments commit

File: learning-development/accessibility.qmd (hunk 2 above)
Comment (cjrace) [reply learning-development/accessibility.qmd:16]: Made this change in my PR comments commit

File: learning-development/accessibility.qmd (hunk 3 above)
Comment (cjrace) [reply learning-development/accessibility.qmd]: Made this change in my PR comments commit

File: learning-development/accessibility.qmd (hunk 4 above)
Comment (cjrace) [reply learning-development/accessibility.qmd]: Made this change in my PR comments commit (great spot!)

File: learning-development/accessibility.qmd (hunk 5 above)
Comment (cjrace) [reply learning-development/accessibility.qmd]: Made this change in my PR comments commit

File: learning-development/accessibility.qmd (hunk 6 above)
Comment (cjrace) [reply learning-development/accessibility.qmd:80]: Made this change in my PR comments commit

File: learning-development/accessibility.qmd [hunk 7]
```diff
@@ -9,10 +9,15 @@ title: "Accessibility"
 
 ## Accessibility in DfE
 
+Everyone has a part to play in ensuring that the content and digital services the DfE provides to it's customers and colleagues are accessible. Everyone should have an understanding of their responsibilities to comply with the [Public Sector Bodies Accessibility Regulations 2018](https://www.legislation.gov.uk/uksi/2018/952/contents/made) and the [Web Content Accessibility Guidelines 2.2](https://www.w3.org/TR/WCAG22/). Moreover, everyone should also understand the important and critical difference this makes to the everyday lives of millions of people.
```
Comment (jen-machin) [inline learning-development/accessibility.qmd:12]: Should be "its customers" not "it's customers" - sorry, very pedantic

File: learning-development/accessibility.qmd [hunk 8]
```diff
@@ -39,7 +44,7 @@ For more advice about colour in charts and visualisations specifically, includin
 
 ## Free tools for reviewing web pages
 
-- [axe DevTools](https://chromewebstore.google.com/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?pli=1), free chrome extension and paid versions, we recommend this (and GDS tend to recommend too)
+- [axe DevTools](https://chromewebstore.google.com/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?pli=1), free Google Chrome extension and paid versions, we recommend this (and GDS tend to recommend too)
```
Comment (jen-machin) [inline learning-development/accessibility.qmd:47]: Sorry - missed this the first time but I'd consider spelling out GDS in full if we haven't already as I'm not sure everyone is familiar with them

File: learning-development/accessibility.qmd (hunk 7 above)
Comment (cjrace) [reply learning-development/accessibility.qmd:12]: Do not apologise for this - it's a mistake that wants fixing!

Comment (cjrace) [reply learning-development/accessibility.qmd]: Updated in PR response 2 commit

**Review by jen-machin — APPROVED** [review]

**Pull request merged**